identity-and-access-management/iam/quickstart.mdx

```diff
@@ -15,13 +14,29 @@ dates:
 
 <Lightbox src="scaleway-iam-concepts.webp" alt="" />
 
-Read our [dedicated page](/identity-and-access-management/iam/reference-content/overview) for a general overview of IAM and how it works.
+Read our dedicated page for a [general overview of IAM](/identity-and-access-management/iam/reference-content/overview) and how it works.
 
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You are the [Owner](/identity-and-access-management/iam/concepts/#owner) of your Organization
 </Message>
 
+## How to activate IAM
+
+If your Scaleway account was created before 5 December 2022, you need to manually activate IAM. IAM is automatically activated for accounts created since this date.
+
+1. Click the **Identity and Access Management** button on the top right of your [Organization dashboard](https://console.scaleway.com/organization).
+    <Lightbox src="scaleway-iam-button.webp" alt=""/>
+    The following screen displays:
+    <Lightbox src="scaleway-iam-splash.webp" alt=""/>
+2. Click **Discover** to start the activation process.
+3. Read the information displayed, and type **MIGRATE** in the box.
+    <Lightbox src="scaleway-iam-activate.webp" alt=""/>
+4. Click **Activate IAM** to finish.
+
+IAM is activated in your account. All existing configuration and functionality is automatically [migrated](/identity-and-access-management/iam/reference-content/migration/) into IAM for you, from the configuration of members and roles, to API keys.
+
```

Comment:
I think this part should be taken out, because everyone now has IAM automatically activated on their account